src/mainviews/create-job.tsx

```diff
@@ -168,44 +175,44 @@ export function CreateJob(props: ICreateJobProps): JSX.Element {
       label: trans.__('Job name'),
```

Comment:
None of these fields, particularly the Environment and Output Formats fields, are showing up in the UI now.